src/services/YotiCallbackProcessor.ts

```diff
@@ -305,11 +304,9 @@ export class YotiCallbackProcessor {
 				  ],
 			  },
 			  restricted: {
-				  user: {
-					  name: documentFields.full_name,
-					  birthDate: documentFields.date_of_birth,
-				  },
-				  [docName]: documentInfo,
+				  name: documentFields.full_name, // TODO, TO be updated with nameParts as part of F2F-1000 ticket.
```

Comment:
is this to be updated?

Reply:
Once we merge in this code, @Caitlin-cooling can get these changes and update this field as art of work being done in F2F-1000?

Reply:
We have decided to merge PR for 1000 in and then rebase this branch and get the changes in for name and then add in missing bits in QA tests as well.